### ISSUE TITLE
Obtaining the value of the adaptive hyperparameter

### ISSUE THREAD

**Issue by cloud99999999** [lululxvi/deepxde#490]
Hi Lu,

Recently, I was trying to use the layer-wise locally adaptive activation function (L-LAAF) as the activation function for my problem. With reference to the source code below:

https://github.com/lululxvi/deepxde/blob/b59a3a3146a08d22333fd315fa3d4292ed020db9/deepxde/nn/activations.py#L27-L28

I would like to ask, how can I extract the value of 'a' for each iteration? I would like to plot the variable 'a' against a fixed number of iterations (~10000 iterations) and observe how it changes.

Thanks in advance!

**Comment by lululxvi** [comment]
If you use TensorFlow 2.x or PyTorch backend, you should be able to directly print a. But I haven't tried.

**Comment by cloud99999999** [comment]
Hi Lu,

Thank you for the reply.

I was able to print out a single value of 'a' by returning the value from the source code and converting it to a numpy value. But I am not able to iteratively get the values at every epoch (e.g the value of 'a' at 50 epochs, 100 epochs and so on). Ideally, I would like to save these values in a list or a .dat file, similar to how the training and loss data is saved. Do you have any suggestions on how I can do this?

**Comment by lululxvi** [comment]
How about first saving `a` to a variable, and then during the training loop, do a printing?
